src/test/java/nextstep/subway/line/LineAcceptanceTest.java

```diff
@@ -35,51 +57,179 @@ void getLines() {
         // given
         // 지하철_노선_등록되어_있음
         // 지하철_노선_등록되어_있음
+        ExtractableResponse<Response> firstLine = createLine("1호선", "navy");
+        ExtractableResponse<Response> secondLine = createLine("2호선", "green");
 
         // when
         // 지하철_노선_목록_조회_요청
+        ExtractableResponse<Response> response = getLine("");
 
         // then
         // 지하철_노선_목록_응답됨
+        assertThat(response.statusCode()).isEqualTo(HttpStatus.OK.value());
         // 지하철_노선_목록_포함됨
+        List<Long> expectIds = Arrays.asList(firstLine, secondLine).stream()
+            .map(this::getIdWithResponse)
+            .collect(Collectors.toList());
+        List<Long> resultIds = getIdsWithResponse(response);
+        assertThat(checkIdsEquals(expectIds, resultIds)).isTrue();
```

Comment:
```suggestion
        // given
        Long firstLineId = mapId(createLine("1호선", "navy"));
        Long secondLineId = mapId(createLine("2호선", "green"));

        // when
        ExtractableResponse<Response> actual = getLines();
        List<Long> actualIds = mapIds(actual);
        
        // then
        assertThat(actual.statusCode()).isEqualTo(HttpStatus.OK.value());
        assertThat(actualIds).containsExactly(firstLineId, secondLineId);
```

containsExactly 를 활용해보는건 어떨까요

Reply:
해당부분 확인하여 적용하도록 변경하였습니다!